uuid/uuid_test.go

```diff
@@ -10,9 +10,9 @@ import (
 	"testing"
 )
 
-const(
-  LegacyUUIDPath = "/go/src/github.com/uuid.py"
-  // legacyUUIDpath = "/usr/bin/uuid.py"
+const (
+	LegacyUUIDPath = "/go/src/github.com/uuid.py"
+	// legacyUUIDpath = "/usr/bin/uuid.py"
```

Comment:
No need for this commented line, let's remove it

Reply:
OK, so I won't merge today, have no more time to do one more round.

Reply:
And actually, I need this commented line, because each time I run it locally - I must use it.

Reply:
Ok, just add a note on it explains that, then you can merge